cookies.js

```diff
@@ -0,0 +1,130 @@
+const cookies = [{
```

Comment:
While probably useful initially this file should be deleted, it isn't used for anything